### PR TITLE
Feature/onboard flow

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Please review!
It's a long one, so I did a breakdown here: 
https://drive.google.com/a/risevision.com/file/d/0B8XS4J7-uR-eSEJra0xqMFF2aEE/view?usp=sharing

Thanks!

**Comment by ghislaineguerin** [comment]
@RV-Adrian do you have a staged version?

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Yep! https://apps-stage-2.risevision.com/
The video should have the main elements to look for. Thx!

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Have you had a chance to review? Thanks

**Comment by ghislaineguerin** [comment]
👍 looks good

File: src/ui-components/onboarding.scss [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    }
+    .onboard-actions .btn {
+        padding: 1px 5px;
+        font-size: 12px;
+        line-height: 1.5;
+        border-radius: 2px;
+    }
+}
+
+// TEMPORARY PROTOTYPE BUTTONS FOR TESTING
```
Comment (RV-Adrian) [inline src/ui-components/onboarding.scss]: @alex-deaconu 
This PR is ready to merge, just wanted to make sure you weren't using the .proto-nav class before I remove it (that blue button to advance the Steps)
Thanks!

Comment (alex-deaconu) [inline src/ui-components/onboarding.scss]: @RV-Adrian nope, not using. Good to remove. Thanks!

**Comment by alex-deaconu** [comment]
Removed the temporary prototype code. As per the discussion with @RV-Adrian this is ready for merge. Thanks.

**Pull request merged**